docs/commands.md

```diff
@@ -6721,11 +6721,23 @@ Example: [http://{{region}}.eureka.url.to.use:8080/eureka-server/v2](http://{{re
 Using {{region}} will make Spinnaker use AWS regions in the hostname to access discovery so that you can have discovery for multiple regions.
  * `--edda`: The endpoint Edda is reachable at. Edda is not a hard dependency of Spinnaker, but is helpful for reducing the request volume against AWS. See [https://github.com/Netflix/edda](https://github.com/Netflix/edda) for more information.
  * `--environment`: The environment name for the account. Many accounts can share the same environment (e.g. dev, test, prod)
+ * `--launching-lifecycle-hook-default-result`: (*Default*: `ABANDON`) Defines the action the Auto Scaling group should take when the lifecycle hook timeout elapses or if an unexpected failure occurs. This parameter can be either CONTINUE or ABANDON. The default value is ABANDON.
+ * `--launching-lifecycle-hook-heartbeat-timeout-seconds`: (*Default*: `3200`) Set the heartbeat timeout for the lifecycle hook. Instances can " +
+          "remain in a wait state for a finite period of time. The default is one hour (3600 seconds).
```

Comment:
The previous line says the default is 3200 seconds, but the documentation says it's 3600.  Needs to be sync'd.